samples/boards/microbit/pong/README.rst

```diff
@@ -0,0 +1,17 @@
+.. _microbit_pong:
+
+BBC micro:bit pong game
+#######################
+
+Overview
+********
+
+Play pong over Bluetooth between two micro:bit devices.
+
+Building
+********
+
+.. code-block:: console
+
+   $ cd samples/boards/microbit/pong
+   $ make
```

Comment:
I extended the overview section with this, since the game is not that complicated (hence not enough text to justify separate section IMO). The text comes as part of the first patch and describes the capabilities of the game once all three patches of this PR have been merged.